docs/make.jl

```diff
@@ -15,5 +15,5 @@ makedocs(
 )
 
 deploydocs(
-    repo = "github.com/juliaastro/skycoords.jl.git"
+    repo = "github.com/JuliaAstro/SkyCoords.jl.git"
```

Comment:
I thought github was case-insensitive

Reply:
https://travis-ci.org/JuliaAstro/SkyCoords.jl/jobs/605204114?utm_medium=notification&utm_source=github_status
Fails from documenter key and the repo slug comparison

Reply:
Right, this is a literal comparison.